backend/spring-routie/src/test/java/routie/routiespace/controller/RoutieSpaceControllerTest.java

```diff
@@ -38,4 +56,114 @@ public void createRoutieSpace() {
         assertThat(expectedHttpStatus).isEqualTo(actualHttpStatus);
         assertThat(locationUrlPattern.matcher(locationUrl).matches()).isTrue();
     }
+
+    @Test
+    @DisplayName("장소 생성에 성공한다")
+    public void createPlace() {
+        // given
+        String requestBody = """
+                {
+                    "name": "스타벅스 강남점",
+                    "address": "서울시 강남구 테헤란로 123",
+                    "stayDurationMinutes": 60,
+                    "openAt": "09:00",
+                    "closeAt": "22:00",
+                    "breakStartAt": "15:00",
+                    "breakEndAt": "16:00",
+                    "closedDays": ["SUNDAY"]
+                }
+                """;
```

Comment:
픽스처 유지보수, 타입 안정성을 위해 아래와 같이 기존 Request 클래스를 사용해보는 건 어떨까요? 😄 

```suggestion
        // given
        PlaceCreateRequest requestBody = new PlaceCreateRequest(
                "스타벅스 강남점",
                "서울시 강남구 테헤란로 123",
                60,
                LocalTime.of(9, 0),
                LocalTime.of(22, 0),
                LocalTime.of(15, 0),
                LocalTime.of(16, 0),
                List.of(DayOfWeek.SUNDAY)
        );
```